profiles/hydra/master/default.nix

```
@@ -100,6 +100,13 @@ in
         }
       '';
     };
+
+    virtualHosts.holofuel-ui = {
```

Usually, `virtualHost` name matches `serverName`:

```suggestion
    virtualHosts.holofuel-demo = {
```